pkg/api/rest/types.go

```diff
@@ -128,7 +128,31 @@ func (nodeStrategy) Validate(obj runtime.Object) errors.ValidationErrorList {
 	return validation.ValidateMinion(node)
 }
 
-// namespaceStrategy implements behavior for nodes
+// eventStrategy implements behavior for events
+type eventStrategy struct {
+	runtime.ObjectTyper
+	api.NameGenerator
+}
+
+// Events is the default logic that applies when creating and updating event objects.
+var Events RESTCreateStrategy = eventStrategy{api.Scheme, api.SimpleNameGenerator}
```

Comment:
I just had a discussion with @bgrant0607 and outcome there was all things are in a namespace except Namespace.